Keep state borders visible

pjsier opened the pull request:
Closes #144. UX/design feedback on this would be good before merging, but I moved the state borders to the top, but still sourced from OpenMapTiles so we don't have to worry about code changes. I also removed the `maxzoom` property and increased the thickness bit on zoom

![state-outlines](https://user-images.githubusercontent.com/8291663/33491764-d9b742b4-d680-11e7-9432-cce876ccf9fa.gif)

Lane commented:
This looks good, I think Noele had mentioned wanting to turn off boundaries for other countries earlier in the project.  Could we do that and still use the OpenMapTiles layer, or should we use our own state boundaries layers?

pjsier commented:
Ah I forgot about that. I'll make an update to this now, it shouldn't be difficult to just switch the source

pjsier commented:
Just updated so that it uses the tile data source for states. I removed other boundaries in general, took out an extra US layer that wasn't getting used, and changed the `style.json` name because I noticed it was still "Natural Earth" and why not

Lane commented:
Looks good to me! Should I merge now or do you want to wait for design feedback?

pjsier commented:
I think it's fine to wait. I pushed it to the prototype link here http://eviction-lab-prototypes.s3-website.us-east-2.amazonaws.com/state-borders
